app.py

```diff
@@ -60,24 +60,45 @@ class MaxContentLengthWarning(APIWarnings):
     pass
 
 load_dotenv()
+
 connection_string_regex = r"^DefaultEndpointsProtocol=https?;.*;FileEndpoint=https://[a-zA-Z0-9]+\.file\.core\.windows\.net/;$"
-connection_string = os.getenv("NACHET_AZURE_STORAGE_CONNECTION_STRING")
+pipeline_version_regex = r"\d.\d.\d"
 
+CONNECTION_STRING = os.getenv("NACHET_AZURE_STORAGE_CONNECTION_STRING")
 
 FERNET_KEY = os.getenv("NACHET_BLOB_PIPELINE_DECRYPTION_KEY")
 PIPELINE_VERSION = os.getenv("NACHET_BLOB_PIPELINE_VERSION")
 PIPELINE_BLOB_NAME = os.getenv("NACHET_BLOB_PIPELINE_NAME")
 
 NACHET_DATA = os.getenv("NACHET_DATA")
-NACHET_MODEL = os.getenv("NACHET_MODEL")
+
+Model = namedtuple(
+    'Model',
+    [
+        'entry_function',
+        'name',
+        'endpoint',
+        'api_key',
+        'inference_function',
+        'content_type',
+        'deployment_platform',
+    ]
+)
 
 try:
     VALID_EXTENSION = json.loads(os.getenv("NACHET_VALID_EXTENSION"))
     VALID_DIMENSION = json.loads(os.getenv("NACHET_VALID_DIMENSION"))
-except TypeError:
+except (TypeError, json.decoder.JSONDecodeError):
     # For testing
     VALID_DIMENSION = {"width": 1920, "height": 1080}
```

Comment:
why these dimensions?

Reply:
It's the standard dimension for an HD image and will act as default on Nachet if  the `NACHET_VALID_DIMENSION` environment variable is not set or is not able to load.

We can always modify these default value if we want bigger or smaller image